plugins/pagerduty/src/components/PagerdutyCard.tsx

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
The casing of the file is not consistent with others. Should be `PagerDutyCard.tsx`